style.css

```diff
@@ -0,0 +1,97 @@
+#burger-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+#burger-bar-2 {
+  margin-top: 4px;
+}
+
+#burger-bar-3 {
+  margin-top: 4px;
+  margin-bottom: 2px;
+}
+
+#collaboration {
```

Comment:
- [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

Reply:
Change request satisfied.